staticd/static_routes.h

```diff
@@ -1,16 +1,14 @@
 /*
- * Static Routing Information header
- * Copyright (C) 2016 Cumulus Networks
+ * STATICd - static routes header
+ * Copyright (C) 2018 Cumulus Networks, Inc.
  *               Donald Sharp
  *
- * This file is part of Quagga.
- *
- * Quagga is free software; you can redistribute it and/or modify it
+ * FRR is free software; you can redistribute it and/or modify it
  * under the terms of the GNU General Public License as published by the
  * Free Software Foundation; either version 2, or (at your option) any
  * later version.
  *
- * Quagga is distributed in the hope that it will be useful, but
```

Comment:
Since we're changing this, wanna replace the whole header?

Reply:
I just removed the files, instead